src/webgl/shaders/sphereMapping.frag

```diff
@@ -0,0 +1,32 @@
+#version 300 es
+#define PI 3.141592
+
+precision highp float;
+
+uniform sampler2D uSampler;
+uniform mat3 uNewNormalMatrix;
+uniform mat3 uCameraRotation;
+uniform mat4 uNewModelViewMatrix;
+uniform mat4 uModelViewMatrix;
+
+in vec2 vTexCoord;
+in vec3 fvNormal;
+in vec3 faNormal;
+in vec3 faPosition;
+
+out vec4 fragColor;
+
+void main() {
+  vec4 viewModelPosition  = uModelViewMatrix  * vec4(faPosition, 1.0);
```

Comment:
I think `uModelViewMatrix` here is going to be made for the filter shader's rectangle, so I'm not sure it's what we want for doing reflections. Does it work if you set `vViewPosition` to all zeros, since we're assuming the position of the camera is negligible, assuming the texture is on an infinitely large sphere? And if so, can we just do `n = normalize(vGlovalNormal.xyz)` directly without doing a `reflect`?